docs/content/contributing/coding-style.md

```diff
@@ -142,7 +142,7 @@ function _setup_postfix_aliases
       [[ ${UNAME} != "${DOMAIN}" ]] && echo "${DOMAIN}" >> /tmp/vhost.tmp
     done < <(grep -v "^\s*$\|^\s*\#" /tmp/docker-mailserver/postfix-virtual.cf || true)
   else
-    _notify 'inf' "Warning '/tmp/docker-mailserver/postfix-virtual.cf' is not provided. No mail alias/forward created."
+    _log 'inf' "Warning '/tmp/docker-mailserver/postfix-virtual.cf' is not provided. No mail alias/forward created."
```

Comment:
```suggestion
    _log 'debug' "'/tmp/docker-mailserver/postfix-virtual.cf' not provided - no mail alias/forward created"
```

Reply:
👍🏼

Reply:
Done in Done in f097b1d08e40259d9d558b79a9875518292a9466